libp2p/muxers/muxer.nim

```diff
@@ -63,3 +63,5 @@ proc new*(
 
   let muxerProvider = T(newMuxer: creator, codec: codec)
   muxerProvider
+
+method getStreams*(m: Muxer): seq[Connection] {.base.} = doAssert false, "not implemented"
```

Comment:
This will also need an implementation for yamux, around here:
https://github.com/status-im/nim-libp2p/blob/af5299f26c195540d3fbe0f1a20742658f8a6220/libp2p/muxers/yamux/yamux.nim#L509

Should be similar to mplex, using `m.channels`